src/Driver/API/ExceptionConverter.php

```diff
@@ -6,7 +6,11 @@
 
 use Doctrine\DBAL\Driver\Exception;
 use Doctrine\DBAL\Exception\DriverException;
+use Doctrine\DBAL\Query;
 
+/**
+ * @internal
```

Comment:
The interface shouldn't be made internal. Driver developers will implement it in their drivers. Only the specific classes can. Although, it won't prevent the changes like this from being the breaking ones.

Reply:
Ok, then it doesn't make much sense. I'll revert this one.